static/scss/partials/main.scss

```diff
@@ -294,9 +294,39 @@ button:active {
     margin: auto auto 0 0;
 }
 
-main,
-table {
+.dashboard-mpp-promo {
```

Comment:
@maxxcrawford @Vinnl I tried to rebase it main and found some merge conflicts and accepted incoming changes - do these `.dashboard-mpp-promo` classes need to be here, or can they be dropped?

Reply:
I don't see them in [current `main`](https://github.com/mozilla/fx-private-relay/blob/main/static/scss/partials/main.scss), so I think they can be dropped?

As for the submodule updates: I think so? I think it means you've committed to `/privaterelay/locales` yourself, which I think probably shouldn't happen, but will also be automatically overwritten by the next update. If so, it shouldn't be a problem.